components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Engine/StandaloneJobOrchestrator.cs

```diff
@@ -170,6 +170,24 @@ IJsonSerializer jsonSerializer
 
                         if (diagInfo.Id == assignedJob.Value.Job.Id) {
                             _publisherDiagnosticInfo.AddOrUpdate(assignedJob.Value.Job.Id, diagInfo);
+
+                            foreach (var entry in _publishedNodesEntries) {
+                                var id = _publishedNodesJobConverter.
+                                    ToConnectionModel(entry, _standaloneCliModel).CreateConnectionId();
+
+                                if (_publisherDiagnosticInfo[diagInfo.Id].Id == id) {
```

Comment:
`_publisherDiagnosticInfo[diagInfo.Id].Id` is the same as just `diagInfo.Id`.

Reply:
changed